plugins/click-outside.ts

```diff
@@ -0,0 +1,38 @@
+import Vue from 'vue'
```

Comment:
We have a similar library [vue-clickaway](https://www.npmjs.com/package/vue-clickaway), can we try to use that library instead of writing a custom one?

Reply:
Yes, @bangjelkoski had mentioned we were using something similar to [v-click-outside](https://www.npmjs.com/package/v-click-outside). I've tried vue-clickaway but that library only works if the dropdown is rendered on top of the anchor element when you click to activate it. In this case the dropdown is rendered slightly below the anchor causing the slippage settings modal to immediately close. I've tried installing v-click-outside but it lacks Typescript definitions. I'm open to suggestions.